src/main/emme/toolbox/assignment/build_transit_scenario.py

```diff
@@ -353,7 +353,7 @@ def find_walk_link(from_line, to_line):
                             if s.allow_boardings])
             link_candidates = []
             for seg in from_line.segments(True):
-                if not s.allow_alightings:
+                if not seg.allow_alightings:
```

Comment:
@JiaXu1024 please double check this.

Reply:
double checked, the update is correct, since line #355 states "for seg in form_line.segments(True):"

Reply:
Thank you. Would fixing this in ABM3_develop have some impact on current results? Should we try? Adding @cundo92 for any additional thoughts regarding this difference in time transfers function.